### PR TITLE
ISPN-11398 Add template for a Protobuf cache

### PR DESCRIPTION
**Pull request by gustavonalle**
https://issues.redhat.com/browse/ISPN-11938

### CODE REVIEW

File: server/runtime/src/main/resources/infinispan-defaults.xml [hunk 1]
```diff
@@ -25,6 +25,11 @@
       <distributed-cache-configuration name="org.infinispan.DIST_SYNC" remote-timeout="17500" configuration="org.infinispan.LOCAL">
          <state-transfer timeout="60000"/>
       </distributed-cache-configuration>
+      <distributed-cache-configuration name="example.PROTOBUF_DIST" remote-timeout="17500" configuration="org.infinispan.LOCAL">
```
Comment (anistor) [inline server/runtime/src/main/resources/infinispan-defaults.xml:28]: Is ```"example.PROTOBUF_DIST"``` the first of this kind? Are we settled on using the ```example```  prefix for these? any other example of ```example```?

Comment (gustavonalle) [reply server/runtime/src/main/resources/infinispan-defaults.xml:28]: https://infinispan.zulipchat.com/#narrow/stream/118645-infinispan/topic/server.20templates

Comment (anistor) [reply server/runtime/src/main/resources/infinispan-defaults.xml:28]: :+1:

**Comment by anistor** [comment]
Thanks @gustavonalle !

**Pull request merged**